### ISSUE TITLE
[object_detection] Python has stopped working when I run model_main.py

### ISSUE THREAD

**Issue by ycui123** [tensorflow/models#4799]
### System information
- **What is the top-level directory of the model you are using**: faster rcnn 101 resnet
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: N
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Windows 10
- **TensorFlow installed from (source or binary)**: source
- **TensorFlow version (use command below)**: tf-gpu 1.8.0rc0
- **Bazel version (if compiling from source)**: N/A
- **CUDA/cuDNN version**: 9.0/v7
- **GPU model and memory**: GTX 1080TI
- **Exact command to reproduce**:

python object_detection/model_main.py --pipeline_config_path=object_detection/faster_rcnn_resnet_101.config --model_dir=object_detection/model/ --num_train_steps=30000 --num_eval_steps=2000

### Describe the problem
Every time I run the command, "python has stopped working" show up. When I ran on pycharm, I got "Process finished with exit code -1073741819 (0xC0000005)". 

### Source code / logs
(debug on MSVS2015)
'python.exe' (Win32): Loaded 'C:\Users\asd\anaconda3\Lib\site-packages\pandas\_libs\testing.cp35-win_amd64.pyd'. Module was built without symbols.
'python.exe' (Win32): Loaded 'C:\Users\asd\anaconda3\Lib\site-packages\matplotlib\_contour.cp35-win_amd64.pyd'. Module was built without symbols.
'python.exe' (Win32): Loaded 'C:\Users\asd\anaconda3\Lib\site-packages\matplotlib\ft2font.cp35-win_amd64.pyd'. Module was built without symbols.
'python.exe' (Win32): Loaded 'C:\Users\asd\anaconda3\Library\bin\freetype.dll'. Module was built without symbols.
'python.exe' (Win32): Loaded 'C:\Users\asd\anaconda3\Library\bin\libpng16.dll'. Module was built without symbols.
'python.exe' (Win32): Loaded 'C:\Users\asd\anaconda3\Lib\site-packages\matplotlib\_png.cp35-win_amd64.pyd'. Module was built without symbols.
'python.exe' (Win32): Loaded 'C:\Users\asd\anaconda3\Lib\site-packages\kiwisolver.cp35-win_amd64.pyd'. Module was built without symbols.
'python.exe' (Win32): Loaded 'C:\Users\asd\anaconda3\Lib\site-packages\matplotlib\_image.cp35-win_amd64.pyd'. Module was built without symbols.
'python.exe' (Win32): Loaded 'C:\Users\asd\anaconda3\Lib\site-packages\matplotlib\_tri.cp35-win_amd64.pyd'. Module was built without symbols.
'python.exe' (Win32): Loaded 'C:\Users\asd\anaconda3\Lib\site-packages\matplotlib\_qhull.cp35-win_amd64.pyd'. Module was built without symbols.
'python.exe' (Win32): Loaded 'C:\Users\asd\anaconda3\Lib\site-packages\matplotlib\backends\_backend_agg.cp35-win_amd64.pyd'. Module was built without symbols.
'python.exe' (Win32): Loaded 'C:\Users\asd\anaconda3\Lib\site-packages\sip.pyd'. Module was built without symbols.
'python.exe' (Win32): Loaded 'C:\Users\asd\anaconda3\Lib\site-packages\PyQt5\QtCore.pyd'. Module was built without symbols.
'python.exe' (Win32): Loaded 'C:\Users\asd\anaconda3\Library\bin\Qt5Core.dll'. Module was built without symbols.
'python.exe' (Win32): Loaded 'C:\Windows\System32\mpr.dll'. Cannot find or open the PDB file.
'python.exe' (Win32): Loaded 'C:\Users\asd\anaconda3\Library\bin\icuin58.dll'. Module was built without symbols.
'python.exe' (Win32): Loaded 'C:\Users\asd\anaconda3\Library\bin\icuuc58.dll'. Module was built without symbols.
'python.exe' (Win32): Loaded 'C:\Users\asd\anaconda3\Library\bin\icudt58.dll'. Module was built without symbols.
The thread 0x29c8 has exited with code 0 (0x0).
Unhandled exception at 0x000000005D0374A9 (python35.dll) in python.exe: 0xC0000005: Access violation reading location 0xFFFFFFFFFFFFFFFF.

**Comment by ycui123** [comment]
Update: Found out that "from object_detection import model_lib" gives me that error message.

**Comment by MonkeyWithAComputer** [comment]
Not sure if you are having the same issue I did, but the import model_lib looked like the cause in my case as well. 
If you previously used the api and had an older object detection api setup you might have left over code from that. Check your python dist-packages for it. Mine is located here:/usr/local/lib/python2.7/dist-packages/object_detection-0.1-py2.7.egg
I ended up renaming that folder to object_detection-0.1-py2.7.egg.backup and it seems to have fixed my issues.
